.gitignore

```diff
@@ -3,4 +3,5 @@
 
 # Django
 *.pyc
-*.sqlite3
\ No newline at end of file
+*.sqlite3
+node_modules
```

Comment:
I think we're already ignoring this in `meshapp/.gitignore`

Reply:
I removed it